deployment/keystone/changeset/internal/deploy.go

```diff
@@ -321,12 +322,14 @@ func ConfigureOCR3Contract(env *deployment.Environment, chainSel uint64, dons []
 		if !ok {
 			return fmt.Errorf("failed to get contract set for chain %d", chainSel)
 		}
-		contract := contracts.OCR3
-		if contract == nil {
-			return fmt.Errorf("no ocr3 contract found for chain %d", chainSel)
+
+		contract, err := contracts.GetOCR3Contract(nil)
+		if err != nil {
+			env.Logger.Errorf("failed to get OCR3 contract: %s", err)
+			return fmt.Errorf("failed to get OCR3 contract: %w", err)
```

Comment:
@MStreet3 How come we need to both log + return an error here? Does the returned error not get bubbled up to the user when executing the changeset?